Taskfile.yml

```
@@ -324,55 +322,45 @@ tasks:
       - pre-commit run -a
 
   run-website:
-    desc: Build & serve the static website.
+    desc: Build & serve the static website for interactive development.
     dir: web/website
     cmds:
       - hugo server
 
-  run-web:
-    desc: Build & serve the website & playground.
-    summary:
-      Note that this only live-reloads the static website; and requires
-      rerunning to pick up playground & book changes.
-    dir: web/website
-    cmds:
-      - task: build-web
-      # Then start web server with rendering to disk, so it picks up the playground files.
-      - hugo server --renderToDisk
-
   run-book:
-    desc: Build & serve the book.
+    desc: Build & serve the book for interactive development.
     dir: web/book
     cmds:
       - mdbook serve --port=3000
 
   run-playground:
-    desc: Build & serve the playground.
+    desc: Build & serve the playground for interactive development.
     dir: web/playground
     cmds:
-      - npm install
+      - task install-playground-dependencies
       - npm start
 
-  run-playground-cached:
-    desc: Build & serve the playground, without rebuilding rust code.
+  install-playground-dependencies:
```

```suggestion
  build-playground:
```

...for full consistency; though `install-playground` would be OK if you prefer

---

I am content with the naming of the steps: the `npm install` (of `install-playground-dependencies`) clearly builds the prerequisites. 

It's hard to see what the `npm start` does - it calls `react-scripts start` which goes into that rabbit hole, so it's hard to say what else that script does to "build" the playground.

Let's just commit this and we can factor further when we have a reason.

---

OK for this thread